src/vaadin-grid-column.html

```diff
@@ -322,6 +322,9 @@
       }
 
       cells.forEach(cell => {
+        if (!cell.parentElement) {
+          return;
```

Comment:
Let's add a comment to briefly mention when this could happen and cause problems.

Reply:
done.